docroot/sites/all/modules/custom/node_stat/node_stat.module

```diff
@@ -0,0 +1,78 @@
+<?php
+
+function node_stat_node_view($node, $view_mode = 'full') {
+  if ($view_mode == 'full') {
+
+    global $user;
+
+    $record = node_stat_get_record($node->nid, $user->uid);
```

Comment:
`$user->uid` is passed as second parameter but is not used in `node_stat_get_record`, probably a bit of code from somewhere else?